src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -101,4 +101,9 @@ public boolean ShooterSpeedIsReady() {
       return shooterFeedback.atSetpoint();
     }
   }
+
+  public boolean atVelocity(double velocitySetpoint) {
+    return inputs.shooterLeftVelocityRadPerSec >= velocitySetpoint
```

Comment:
Not sure if it would make a big difference, but should this allow a delta value (e.g. `inputs.shooterLeftVelocityRadPerSec >= velocitySetpoint - 0.05`) to make it notice the goal sooner?